app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -48,8 +52,25 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
               className="w-full flex-grow-0"
               label="Description"
               name="description"
-              rows={8}
+              rows={1}
             />
+            <div className="w-full flex-col justify-between">
+              <Select
+                required
+                label="Assigned Contact"
+                name="contactList"
+                options={CONTACT_OPTIONS}
+                placeholder="Select Contact"
+              />
+              <Select
+                required
+                className="pt-4"
+                label="Tags"
+                name="tagList"
+                options={TAG_OPTIONS}
+                placeholder="Select tag"
+              />
```

Comment:
Why is that in your business logic you are not allowing the user to add multiple tags to a note? We should be allowed to choose multiple tags.

Reply:
I was expecting each note to be under one tag. 
I have added the functionality to chose multiple tags.